app/controllers/agenda-items/agenda-item/session.ts

```diff
@@ -1,22 +1,16 @@
 import Controller from '@ember/controller';
 import { ModelFrom } from '../../../lib/type-utils';
 import { sortObjectsByTitle } from 'frontend-burgernabije-besluitendatabank/utils/array-utils';
-import AgendaItem from 'frontend-burgernabije-besluitendatabank/models/agenda-item';
 import AgendaItemsAgendaItemSessionRoute from 'frontend-burgernabije-besluitendatabank/routes/agenda-items/agenda-item/session';
 
 export default class AgendaItemsAgendaItemSessionController extends Controller {
   /** Used to fetch agenda items from the model */
   declare model: ModelFrom<AgendaItemsAgendaItemSessionRoute>;
 
   get agendaItemsSorted() {
-    let agendaItems = this.model.session.hasMany('agendaItems').value() as
-      | AgendaItem[]
-      | null;
-
-    if (!agendaItems) {
-      agendaItems = [];
-    }
-
-    return agendaItems.slice().sort(sortObjectsByTitle);
+    return this.model.agendaItemOnSameSession
+      ?.concat(this.model.agendaItem)
```

Comment:
**Issue:**  `concat` create a duplicate entry
![image](https://github.com/lblod/frontend-burgernabije-besluitendatabank/assets/3050307/88aadb32-0c88-4df5-8d98-f39e08ce1a7e)

Reply:
Oh strange. I added the concat because it seemed the 'original' agenda item was missing. But that must be because the filtering is now in the controller. 
Will be fixed :)